main/mainTemplate.json

```diff
@@ -110,7 +117,7 @@
     },
     "virtualNetworkAddressPrefix": {
       "type": "string",
-      "defaultValue": "10.0.0.0/8",
+      "defaultValue": "10.0.0.0/16",
```

Comment:
I think We need to remove defaultValue of Virtual Network and subnet in mainTemplate as well.

Reply:
Done.